features/style-attributes.yml

```
@@ -0,0 +1,5 @@
+name: Style attributes
```

```suggestion
name: Style attribute
```

---

I'd like to go through all the attr features and harmonize the way we're doing this (some are the attr name alone, some are like "translate attribute", some are like this). I suspect most of them should be in the style of `attrname (attribute)` when they disambiguate themselves from something else (like this does, for `<style>`):

```suggestion
name: style (attribute)
```